### ISSUE TITLE
Implement  FileListWidget.FILES_AND_DIRECTORIES also for FileWidget

### ISSUE THREAD

**Issue by tischi** [scijava/scijava-ui-swing#83]
It would be great to implement `FileListWidget.FILES_AND_DIRECTORIES` also for `FileWidget`.

See forum post for details: https://forum.image.sc/t/scijava-ui-open-both-file-and-directory/97389/4

**Comment by tischi** [comment]
Hi @imagejan, I had a brief look and this should be doable...do you want me to create a PR (I may have some time in a train on Sunday...)?

**Comment by imagejan** [comment]
@tischi yes, please do if you find the time. I'll be happy to review and test!

**Comment by tischi** [comment]
@imagejan here we go: https://github.com/scijava/scijava-ui-swing/pull/84
